extensions/wikia/UserPreferencesV2/UserPreferencesV2.class.php

```diff
@@ -396,8 +396,8 @@ static public function onSpecialPreferencesBeforeResetUserOptions( $preferences,
 		$userIdentityObject = new UserIdentityBox( $user );
 		$mastheadOptions = $userIdentityObject->getFullData();
 		$masthead = new Masthead( $user );
-		if ( !empty( $masthead->mUser->mOptionOverrides['avatar'] ) ) {
-			$mastheadOptions['avatar'] = $masthead->mUser->mOptionOverrides['avatar'];
+		if ( !empty( $masthead->mUser->getOptionOverrides()['avatar'] ) ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "avatar" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)